### PR TITLE
Removed Mongo DB and Keystone

### PR DESCRIPTION
**Pull request by Raistlfiren**
Updated code base to remove mongo and keystone. It seems to work just fine. Maybe, sort of, kind of...

### CODE REVIEW

**Comment by Raistlfiren** [comment]
@Blodir Please test now. Thanks :)

**Comment by Raistlfiren** [comment]
bump @Blodir

File: .env [hunk 1]
```diff
@@ -3,4 +3,5 @@ NODE_ENV=development
 WP_URL=http://faforever.com/wp-json
 
 CHALLONGE_USERNAME=joe
-CHALLONGE_APIKEY=12345
\ No newline at end of file
+CHALLONGE_APIKEY=12345
+PORT=4000
```
Comment (Cheyans) [inline .env]: Newline

File: express.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	//500 Error Handler
+	app.use(function (err, req, res, next) {
+		res.status(500).render('errors/500', {error: err});
+	});
+}
+
+//Start and listen on port
+app.listen(app.get('port'), function () {
+	console.log('Express listening on port ' + app.get('port'));
+});
```
Comment (Cheyans) [inline express.js]: Newline

File: routes/middleware.js [hunk 3]
```diff
@@ -33,44 +20,6 @@ exports.initLocals = function(req, res, next) {
...
-
-	if (!req.user) {
-		req.flash('error', 'Please sign in to access this page.');
-		res.redirect('/keystone/signin');
-	} else {
-		next();
-	}
-
-};
+};
```
Comment (Cheyans) [inline routes/middleware.js]: Newline

File: templates/views/errors/500.pug [hunk 4]
```diff
@@ -3,4 +3,5 @@ extends ../../layouts/default
 block content
 	.container
 		h1 Error (500)
-		p.lead Sorry, the site has encountered an error.
\ No newline at end of file
+		p.lead Sorry, the site has encountered an error.
+		pre #{error.stack}
```
Comment (Cheyans) [inline templates/views/errors/500.pug]: Newline

File: Dockerfile [hunk 5]
```diff
@@ -16,8 +12,8 @@ RUN npm install
 ADD . /code/
 
 RUN npm install -g grunt-cli
-RUN grunt prod
+RUN grunt serve
```
Comment (Cheyans) [inline Dockerfile]: Why did you change this to the dev build?

File: express.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
+// Simulate config options from your production environment by
+// customising the .env file in your project's root folder.
+require('dotenv').load();
```
Comment (Cheyans) [inline express.js]: Use process.env.ENVIRONMENT_VARIABLE

File: templates/views/blog.pug [hunk 7]
```diff
@@ -3,7 +3,7 @@ extends ../layouts/default
 block css
 	link(href="/styles/css/news.min.css", rel="stylesheet")
 
-// MIXINS
+//- MIXINS
```
Comment (Cheyans) [inline templates/views/blog.pug]: why did you ad `-`

File: templates/views/blog.pug [hunk 8]
```diff
@@ -33,7 +33,7 @@ mixin post(post)
 								if type_index < post._embedded["wp:term"].length - 1 && post._embedded["wp:term"][1].length > 0
 									| ,&nbsp;
 
-// CONTENT
+//- CONTENT
```
Comment (Cheyans) [inline templates/views/blog.pug]: see above?

File: .env (hunk 1 above)
Comment (Blodir) [inline .env]: Port is 4000, but dockerfile exposes 3000

**Comment by Blodir** [comment]
You changed the port to 4000, but Dockerfile still exposes 3000

Admin Login & "Powered by Keystone" can now be removed from footer

File: express.js (hunk 6 above)
Comment (Raistlfiren) [inline express.js]: I guess that would be one less dependency. I will look into it.

File: templates/views/blog.pug (hunk 7 above)
Comment (Raistlfiren) [inline templates/views/blog.pug]: PUG won't compile correctly without the `-`. A bug with pug.

**Comment by Raistlfiren** [comment]
okay @Blodir I have fixed the issues and updated the README.
